frontend/src/App.tsx

```diff
@@ -1,14 +1,28 @@
 import React from "react";
-// import logo from "./logo.svg";
+import { BrowserRouter as Router, Route, Routes } from "react-router-dom";
+import Navbar from "./components/navbar";
+import Home from "./Home";
+import Logout from "./Logout";
+import ProcessedDataPage from "./pages/processedDataPage";
+import PredictedFiles from "./PredictedFiles";
 import "./App.css";
-import { Outlet } from "react-router";
 
-function App() {
+function App(): React.ReactElement<typeof Router> {
   return (
-    // navbar component goes above outlet
-    <div>
-      <Outlet />
-    </div>
+    <Router>
+      <Navbar />
+      <Routes>
+        {/* Public Routes */}
+        <Route path="/" element={<Home />} />
+        <Route path="/Logout" element={<Logout />} />
+
+        {/* Protected Routes */}
+        {/* This tag is to be added when the implementation of authentication is to be created */}
+        {/* <Route element={<RequireAuth type={"user"} />}></Route> */}
```

Comment:
I know I made you remove my FileUpload Page earlier but could you link the route for it here as well since your branch now contains my changes